### ISSUE TITLE
Possibility of disabling progress bar

### ISSUE THREAD

**Issue by ruzickap** [TheFox/flickr-cli#38]
Is there a way how to disable the progress bar? 

If I run something like with redirection to file:

```bash
(
docker run --rm -it -u $(id -u):$(id -g) -v "$PWD":/mnt -v "$HOME/.flickr-cli":/data thefox21/flickr-cli upload --recursive --config=/data/config.yml my_pictures
) 2>&1 | tee -a flickr_upload.log
```

The output saved to `flickr_upload.log` looks terrible, because of the progress bar.

Any change to introduce some parameter which may disable it?

Thank you

**Comment by samwilson** [comment]
Good idea. I think the progress bar should only be displayed in level-2 verbose (`-vv`) mode.

**Comment by ruzickap** [comment]
Actually I like progress bar "feature" - it's handy when you are uploading few files.

When you are uploading thousands of photos you usually use some scripting + store the output to file to check the errors/results later.

I suggest to keep the progress bar like it is now, but maybe introduce parameter like `--noprogress` which will disable it.

(or introduce some new parameter which forces `flickr-cli` output to be more "file output" friendly)

**Comment by TheFox** [comment]
I like the `--noprogress` idea.
